README.md

```diff
@@ -211,13 +202,85 @@ Sigstore instance options:
   --rekor-root-pubkey FILE
                         A PEM-encoded root public key for Rekor itself
                         (conflicts with --staging) (default: None)
+  --certificate-chain FILE
+                        Path to a list of CA certificates in PEM format which
+                        will be needed when building the certificate chain for
+                        the Fulcio signing certificate (default: None)
 ```
 <!-- @end-sigstore-verify-identity-help@ -->
 
 For backwards compatibility, `sigstore verify [args ...]` is equivalent to
 `sigstore verify identity [args ...]`, but the latter form is **strongly**
 preferred.
 
+#### Signatures from GitHub Actions
```

Comment:
Might be nice to show an example of this, maybe by verifying our releases?

Reply:
Agreed! I'm thinking about adding a separate "Quickstart" or "Cookbook" section to the README for that, if that sounds good to you.